### PR TITLE
EITI Webfont

### LINKED ISSUE #724: BUG Glossary book icon alignment
The alignment is currently
screenshot 2015-11-09 08 42 45

It should be
screenshot 2015-11-09 08 44 08

### PR DESCRIPTION
**Pull request by gemfarmer**
This PR addresses #718, fixes #724 

Preview URL : https://federalist.18f.gov/preview/18F/doi-extractives-data/webfont/

What it does: 
* Adds additional icons to the growing catalog of Font Family 'eiti'
* documents said fonts in styleguide [here](https://federalist.18f.gov/preview/18F/doi-extractives-data/webfont/styleguide/section-webfont.html)
* replaces existing SVGs and PNGs with webfonts, styles them accordingly (for a more comprehensive listing, see #718 )

Assigning to @meiqimichelle, but could be handled by someone else if you'd prefer :)

** before anything gets merged, I'm going to address the hound suggestions

### CODE REVIEW

File: _sass/mixins/_rotate.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@mixin rotate($degrees, $rotation) {
+  filter: progid:DXImageTransform.Microsoft.BasicImage(rotation=#{$rotation});
+  -webkit-transform: rotate($degrees);
+      -ms-transform: rotate($degrees);
```
Comment (gemfarmer) [inline _sass/mixins/_rotate.scss]: I think this is an exception to the suggested formatting, so I'm keeping.

**Comment by meiqimichelle** [comment]
@gemfarmer This looks great! There might be some minor adjustments from Eric on relative sizing between the icons, but we can get that in style cleanup later. I was going to merge, but I see you have a note about addressing some hound errors. I'm re-assigning to you to merge when you've done what you want with hound.

**Comment by meiqimichelle** [comment]
@gemfarmer One more thing tho -- I didn't see that this was also meant to resolve #724 . This would not close out that issue (there are still some adjustments that need to be made -- the book is slightly too large still and the padding needs to be smaller on the left, and greater on the right) -- but I recommend merging this and dealing with the visual bug in a separate PR.

**Comment by gemfarmer** [comment]
@meiqimichelle Ok. I see what you mean. The sizing is still slightly off. I'll do as you recommend – merge and address in separate ticket :)

**Pull request merged**